<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgFCCSResetSetParameter</class>
 <widget class="QDialog" name="DlgFCCSResetSetParameter">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>244</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set/Reset Fit Pramater</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="btnHelp">
     <property name="text">
      <string>&amp;Help</string>
     </property>
     <property name="icon">
      <iconset resource="../../lib/lib.qrc">
       <normaloff>:/lib/help/help.png</normaloff>:/lib/help/help.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Parameter:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cmbParam">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Properties:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QWidget" name="widget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QRadioButton" name="radValueDontChange">
            <property name="text">
             <string>don't change</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radValueReset">
            <property name="text">
             <string>reset</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radValueSet">
            <property name="text">
             <string>set value:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QFDoubleEdit" name="edtValue">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QWidget" name="widget_1" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QRadioButton" name="radErrorDontChange">
            <property name="text">
             <string>don't change</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radErrorReset">
            <property name="text">
             <string>reset</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radErrorSet">
            <property name="text">
             <string>set value:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QFDoubleEdit" name="edtError">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QRadioButton" name="radFixDontChange">
            <property name="text">
             <string>don't change</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radFixReset">
            <property name="text">
             <string>reset</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radFixSet">
            <property name="text">
             <string>set value:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QCheckBox" name="chkFixed">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>fixed</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Options:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="chkChangeAll">
       <property name="text">
        <string>change in all RDRs</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="chkInitial">
       <property name="text">
        <string>set in initial parameters</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QCheckBox" name="chkResetAllIndices">
       <property name="text">
        <string>set all runs/indices/pixels</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QFDoubleEdit</class>
   <extends>QLineEdit</extends>
   <header>qfdoubleedit.h</header>
   <slots>
    <signal>valueChanged(double)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../lib/lib.qrc"/>
  <include location="../../lib/lib.qrc"/>
  <include location="../../lib/lib.qrc"/>
  <include location="../../lib/lib.qrc"/>
  <include location="../../lib/lib.qrc"/>
  <include location="../../lib/lib.qrc"/>
  <include location="../../lib/lib.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DlgFCCSResetSetParameter</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>367</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DlgFCCSResetSetParameter</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>435</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radValueSet</sender>
   <signal>toggled(bool)</signal>
   <receiver>edtValue</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>525</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radErrorSet</sender>
   <signal>toggled(bool)</signal>
   <receiver>edtError</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>525</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radFixSet</sender>
   <signal>toggled(bool)</signal>
   <receiver>chkFixed</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>525</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edtValue</sender>
   <signal>textChanged(QString)</signal>
   <receiver>DlgFCCSResetSetParameter</receiver>
   <slot>dataEdited()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>505</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>532</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edtError</sender>
   <signal>textChanged(QString)</signal>
   <receiver>DlgFCCSResetSetParameter</receiver>
   <slot>dataEdited()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>467</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>536</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkFixed</sender>
   <signal>toggled(bool)</signal>
   <receiver>DlgFCCSResetSetParameter</receiver>
   <slot>dataEdited()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>489</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>509</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>dataEdited()</slot>
 </slots>
</ui>
